scripts/cleanup-test-processes.sh

```diff
@@ -7,5 +7,16 @@ pkill --echo --uid "${UID}" --full '/ganache.* --deterministic '
 
 # If you have monero-wallet-rpc or swapd processes owned by the current user
 # that you don't want to kill, don't use this script!
+if pkill --echo --uid "${UID}" --full '/swapd '; then
+	# If swapd instances were killed, give the monero-wallet-rpc instances
+	# some time to shutdown.
+	sleep 4 # Give time for wallet to shutdown
+fi
+
+# Take note of monero-wallet-rpc instances being killed here if the instances
+# were started by swapd
 pkill --echo --uid "${UID}" --full '/monero-wallet-rpc '
```

Comment:
At least on Linux, this has been working pretty good. After I added the short sleep after killing `swapd` instances, I haven't seen any `monero-wallet-rpc` instances that were both created by `swapd` but not shut down when `swapd` exited. Vetting the `monerod` node before starting `monero-wallet-rpc` was also critical to getting this working.  `monero-wallet-rpc` had incredibly slow shutodown times when it could not communicate with the monero node.